pkg/chain/bitcoin/electrs.go

```diff
@@ -64,10 +61,10 @@ func (e ElectrsConnection) Broadcast(transaction string) error {
 	return nil
 }
 
-// VbyteFee retrieves the 25-block estimate fee per vbyte on the bitcoin network.
-func (e ElectrsConnection) VbyteFee() (int32, error) {
+// VbyteFeeFor25Blocks retrieves the 25-block estimate fee per vbyte on the bitcoin network.
+func (e ElectrsConnection) VbyteFeeFor25Blocks() (int32, error) {
```

Comment:
Small note here: `ElectrsConnection` may not need to be exported. If we do a `func Connect(url string) (Handle, error)`, it can return whatever internal handle is appropriate based on the URL.

Reply:
@beaushinkle what we decided to do about this comment? I can see we've defined a `bitcoin.Handle`, can we return it from `NewElectrsConnection` function and make `ElectrsConnection` non-exported?